### ISSUE TITLE
S-MultiXcan: extracting exact z-scores used to compute p-values (for p < e-324)

### ISSUE THREAD

**Issue by DoraCChen** [hakyimlab/MetaXcan#170]
Hello MetaXcan authors, 

I wanted to ask if there was any way to manually compute or extract the exact Z-score used to compute p-values for genes with p-values smaller than the smallest possible floating point number in python (https://github.com/hakyimlab/MetaXcan/issues/111)? Looking at the S-MultiXcan documentation, I do see that Z-score min, max, mean, and SD values are outputted; however, the raw z-score doesn't seem to be provided. Thanks!

z_min: minimum z-score among single-tissue S-PrediXcan associations.
z_max: maximum z-score among single-tissue S-PrediXcan associations.
z_mean: mean z-score among single-tissue S-PrediXcan associations.
z_sd: standard deviation of the mean z-score among single-tissue S-PrediXcan associations.
tmi: trace of T * T', where This correlation of predicted expression levels for different tissues multiplied by its SVD pseudo-inverse. It is an estimate for number of indepent components of variation in predicted expresison across tissues (typically close to n_indep)
status: If there was any error in the computation, it is stated here

**Comment by Heroico** [comment]
> MetaXcan doesn’t calculate a Z-score, just the p-value.

I think there's a typo here; I'd like to clarify that MultiXcan/S-MultiXcan can't compute a z-score.

**Comment by hakyim** [comment]
Yes, good point. Thank you for spotting the error, Alvaro. I meant to say MultiXcan doesn’t calculate z-score.
